lib/bcdice/game_system/SwordWorld2_5.rb

```diff
@@ -119,6 +184,23 @@ def druid_dice(command, power_list)
 
         return sequence.join(" ＞ ")
       end
+
+      def get_abyss_curse_table
+        table_result = self.class::ABYSS_CURSE_TABLE.roll(@randomizer)
+        additional =
+          case table_result.value
+          when 14  # 「差別の」における分類決定表
+            self.class::ABYSS_CURSE_CATEGORY_TABLE.roll(@randomizer).to_s
+          when 25  # 「過敏な」における属性決定表
+            self.class::ABYSS_CURSE_ATTR_TABLE.roll(@randomizer).to_s
+          end
```

Comment:
```suggestion
        table_result = ABYSS_CURSE_TABLE.roll(@randomizer)
        additional =
          case table_result.value
          when 14  # 「差別の」における分類決定表
            ABYSS_CURSE_CATEGORY_TABLE.roll(@randomizer).to_s
          when 25  # 「過敏な」における属性決定表
            ABYSS_CURSE_ATTR_TABLE.roll(@randomizer).to_s
          end
```

クラスごとに設定するわけではないので `self.class::` は不要です。